tests/unit/converters/test_pandapower_converter_input.py

```diff
@@ -134,6 +139,7 @@ def test_parse_data__extra_info(
     fill_pp_extra_info_mock.assert_called_once_with(extra_info=extra_info)
 
 
+@pytest.mark.xfail()
```

Comment:
todo: i'm guessing you just haven't updated the test yet, which is fine, hence the todo 😄